microcosm_postgres/encryption/api.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import event
+
+from microcosm_postgres.encryption.encryptor import Encryptor
+
+
+def init_encryption(encryptor: Encryptor, model_class, key: str) -> None:
```

Comment:
There's room to make a lot of this initialization more object-oriented and more automatic:

 - I'm hardcoding the field "plaintext" and passing in the encryption context string. Both of these ought to be derived from the `model_class`
 - I'm intentionally not mapping the encryptor to a microcosm `binding` yet, so this init call is also manually invoked (see unit test).